hw/ip/sram_ctrl/doc/dv/index.md

```diff
@@ -86,10 +136,62 @@ The following covergroups have been developed to prove that the test intent has
 #### Scoreboard
 The `sram_ctrl_scoreboard` is primarily used for end to end checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
+* tl_a_chan_fifo: TL address channel for CSR accesses
+* tl_d_chan_fifo: TL data channel for CSR accesses
+* alert_fifos: Alert handshakes
+* sram_tl_a_chan_fifo: TL address channel for memory accesses
+* sram_tl_d_chan_fifo: TL data channel for memory accesses
+* kdi_fifo: For key refresh operations from OTP
 <!-- explain inputs monitored, flow of data and outputs checked -->
 
+All CSR accesses made to the SRAM_CTRL register file are tracked and predicted accordingly inside the scoreboard.
+
+Verifying memory accesses is a little more involved of a process, and makes
+heavy use of the [`mem_bkdr_if`]({{< relref "hw/dv/sv/mem_bkdr_if/mem_bkdr_if.sv" >}}).
+
+At a high level, whenever a TL access is made to memory, all relevant information is stored in a transaction-level struct
+(such as write_enable, address, write data, mask, key, and nonce).
```

Comment:
It is unclear if these are all or some of the fields, or why these specific ones are mentioned.

Reply:
that's a good point - I'll add a brief description of the actual struct here.